mitmproxy/addons/maplocal.py

```diff
@@ -0,0 +1,135 @@
+import mimetypes
+import re
+import typing
+from pathlib import Path
+
+from werkzeug.security import safe_join
+
+from mitmproxy import ctx, exceptions, flowfilter, http
+from mitmproxy.addons.modifyheaders import parse_spec
```

Comment:
Should we move `parse_spec` to a utility or something? It's semi-odd to import something from an unrelated addon, or is it just me?

Reply:
Ok, let's move this to `mitmproxy/utils/spec.py`

Reply:
Good idea, done.